src/app/operations/globals.rs

```diff
@@ -2,8 +2,15 @@ use crate::app::App;
 
 impl App {
     pub async fn globals(&self) -> anyhow::Result<()> {
-        let globals = self.client.globals().await?;
-        println!("{}", serde_json::to_string_pretty(&globals)?);
-        Ok(())
+        match self.client.globals().await {
+            Ok(globals) => {
+                println!("{}", serde_json::to_string_pretty(&globals)?);
+                Ok(())
+            }
+            Err(err) => {
+                println!("Error occurred while fetching globals");
```

Comment:
if you are going to print in the error branch you should use `eprintln`.

But when using `anyhow` I think you can reduce the code to:
```
let globals = self.client.globals().await.context("Error while fetching globals")?
```

Reply:
The intent here was to produce better formatted error outputs. However, I observe that it doesn't make a significant difference. Replaced result matching with `.context` in 89c688ef6e37082aa02359d5e93497d1a0a19634 and 3b777a5797fa8e45d7b8a3a9326f6ccdfa8d7c88